<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Составление расписания выполнения зависимых процессов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="edgesInputTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <row/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="inputGraphLbl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ввод графа</string>
    </property>
   </widget>
   <widget class="QLabel" name="vertexCountLbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество вершин</string>
    </property>
   </widget>
   <widget class="QLabel" name="edgesCountLbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество рёбер</string>
    </property>
   </widget>
   <widget class="QLabel" name="edgesInputLbl">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ввод рёбер</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vertexCountEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edgesCountEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="typeCountEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="processorCountLbl">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество процессоров по типам</string>
    </property>
   </widget>
   <widget class="QLabel" name="scheduleLbl">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Порядок выполнения работ</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="taskTypeTable">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <row/>
    <column/>
   </widget>
   <widget class="QLabel" name="taskTypeLbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип задачи</string>
    </property>
   </widget>
   <widget class="QLabel" name="typeCountLbl">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество типов</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="processorCountTable">
       <property name="rowCount">
        <number>1</number>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <row/>
       <column/>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="graphInputBtn">
       <property name="text">
        <string>Ввод данных</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
